### DISCUSSION TITLE
[Feature] Use ML for identifying all similar photos for an album or updating date/location

### DISCUSSION THREAD

**Discussion by ttufts** [immich-app/immich#10908] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a lot of photos with incomplete/inconsistent/no EXIF data that I'm currently loading into my immich database. This results in tons of filtering through photos that have no date and trying to pick "oh these are all from my wedding photo directory" or "oh these are from that trip to Italy". I am no ML expert but I imagine it would be possible to identify similar photos that were taken with similar background/foreground/lighting or where the people have the same clothing on. 

This would be a great feature for migrating to Immich as sorting through loads of photos all at once is daunting. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Covered by #1673/#4509
